Modules/Package.swift

```diff
@@ -86,6 +86,7 @@ let package = Package(
         .target(
             name: "WordPressShared",
             dependencies: [
+                .product(name: "CocoaLumberjackSwift", package: "CocoaLumberjack"),
```

Comment:
Not too happy with having to put CocoaLumberjack in WordPressShared. It feels like that modules should be more lower level and don't worry about logging.

"Had to" to compensate for lacking error logging — adding Tracks would have been more complex and out of boundary!

Reply:
If you agree, we might leave a note to revisit this after the extraction push.

```suggestion
                // TODO: Consider removing CocoaLumberjack from this lower level framework
                .product(name: "CocoaLumberjackSwift", package: "CocoaLumberjack"),
```

Reply:
I'd keep the TODO:. `WordPressShared` is the module that everything depends on, so it needs to compile quickly and have as minimum dependency as possible, or it risks becoming a bottleneck. 

The logging system would ideally be defined as a protocol-module with the implementing being injected by the app. This way `CocoaLumberjackSwift` could be compiled in parallel with everything else.

It could be premature optimization. The best way to know is to analyze the Build Timing after we complete the main efforts to identity the actual bottlenecks.

Reply:
I ended up removing CocoaLumberjack because of a crash in the tests due to, my guess, linking.

Opened to https://github.com/wordpress-mobile/WordPress-iOS/issues/24252 keep this in mind.